libraries/networking/src/NetworkingConstants.h

```diff
@@ -83,8 +83,8 @@ namespace NetworkingConstants {
 }
 
 const QString HIFI_URL_SCHEME_ABOUT = "about";
-const QString URL_SCHEME_HIFI = "hifi";
-const QString URL_SCHEME_HIFIAPP = "hifiapp";
+const QString URL_SCHEME_VIRCADIA = "hifi";
+const QString URL_SCHEME_VIRCADIAAPP = "hifiapp";
```

Comment:
We're renaming the variable but keeping the contents as "hifi"?

Reply:
Yeah, the changing of the variable will happen in the other PR I have open.